pkg/runner/builtin/runner.go

```diff
@@ -240,7 +240,27 @@ func (r *Runner) Run(w *queue.Work) error {
 }
 
 func (r *Runner) Cancel(job *types.Job) error {
-	client, err := dockerclient.NewDockerClient(DockerHost, nil)
+	var tlc *tls.Config
```

Comment:
I think about this (3 times repeated =)), may be we need something like newDockerClient() function...